eva_submission/biosample_submission/biosamples_submitters.py

```diff
@@ -196,19 +219,20 @@ def submit_biosamples_to_bsd(self, samples_data):
         Then it returns a map of sample name to sample accession.
         """
         for sample in samples_data:
-            sample.update(self.default_communicator.communicator_attributes)
             if self.can_create(sample):
+                sample.update(self.default_communicator.communicator_attributes)
```

Comment:
@tcezard I get the tests to pass locally if we restore this line to where it was before